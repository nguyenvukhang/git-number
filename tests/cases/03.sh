# staged file (number)

init 2
touch file:gold
__gitnu__ status
__gitnu__ add 1-2 # use number
__gitnu__ status
__gitnu__ reset 1 # use number again, on file:gold
save __gitnu__ status

# --------------------------------------------------------------------
# On branch main
#
# No commits yet
#
# Changes to be committed:
# 1	[32mnew file:   file_0001[m
#
# Untracked files:
# 2	[31mfile:gold[m
# 3	[31mfile_0002[m
#
