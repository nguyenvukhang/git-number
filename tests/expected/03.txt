On branch main

No commits yet

Changes to be committed:
1	[32mnew file:   file_0001[m

Untracked files:
2	[31mfile:gold[m
3	[31mfile_0002[m

